### PR TITLE
Update social tags for /oceans

### PR DESCRIPTION
**Pull request by Erin007**
based on copy in this [doc](https://docs.google.com/document/d/1HTZrk7BhVJk4uw5i4P-28QoXpZgNBGZ_ndCsYP2k03Q/edit)

### CODE REVIEW

**Comment by breville** [comment]
Is the new image `oceans.png` actually 1200x630?

**Comment by Erin007** [comment]
> Is the new image `oceans.png` actually 1200x630?

It is now!

**Comment by Erin007** [comment]
Screen Shot 2019-11-26 at 3 30 53 PM

**Review by breville — APPROVED** [review]

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 1]
```diff
@@ -31,6 +31,9 @@
   oceans_landing_cs_for_good: 'CS For Good'
   oceans_landing_cs_for_good_desc: 'Resources to help students understand the role computer science could play in creating a more equitable world.'
 
+  social_hoc2019_oceans_title: 'AI for Oceans #CSForGood'
+  social_hoc2019_oceans_desc: "Learn about AI, machine learning, training data, and bias, while exploring ethical issues and how AI can be used to address world problems. Computer science is about so much more than coding! Enjoy Code.org's first step in a new journey to teach more about AI throughout our curriculum."
```
Comment (breville) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml:35]: Oh I'm curious if this string will still fit everywhere okay?  It seems a bit long?

Comment (Erin007) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml:35]: That's the copy Kirsten gave me. If it needs to be shortened we can do it in a follow-up?

**Pull request merged**